frontend/packages/dev-console/src/components/topology/components/const.ts

```diff
@@ -7,6 +7,10 @@ export const TYPE_SERVICE_BINDING = 'service-binding';
 export const TYPE_APPLICATION_GROUP = 'part-of';
 export const TYPE_TRAFFIC_CONNECTOR = 'traffic-connector';
 export const LAST_TOPOLOGY_VIEW_LOCAL_STORAGE_KEY = `${STORAGE_PREFIX}/last-topology-view`;
+export const TOPOLOGY_LAYOUT_LOCAL_STORAGE_KEY = `${STORAGE_PREFIX}/topology-layout`;
+
+export const LAST_TOPOLOGY_VIEW_CONFIG_STORAGE_KEY = 'devconsole.topology.last-topology-view';
+export const TOPOLOGY_LAYOUT_CONFIG_STORAGE_KEY = 'devconsole.topology.topology-layout';
```

Comment:
It will be good to put `devconsole.topology` in the separate consts to avoid string repetition.